Sources/SpeziOnboarding/ConsentView/ConsentDocumentIdentifier.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  File.swift
+//  
+//
+//  Created by Patrick Langer on 03.08.2024.
+//
+
+import Foundation
+
+/// A type representing an unique identifier for a `ConsentDocument`. Allows to distinguish multiple documents during `ConsentConstraint.store`.
+public struct ConsentDocumentIdentifier: Identifiable, Sendable, Equatable {
```

Comment:
Would be great to learn your reasoning about the wrapper type here. As far as I can see working directly with a string might make things a bit easier (also easier for a Codable conformance and more) but happy to be convinced otherwise 👍

Reply:
I agree, it is not really needed. I have reverted it back to Strings :)
My initial thought was to make it more strictly typed and adhere to the Identifiable protocol, but I agree that simple Strings make it easier.